functions/alarm/CreateAlarm.cs

```diff
@@ -0,0 +1,34 @@
+using System.IO;
+using System.Threading.Tasks;
+using AlarmSystem.Core.Entity.Dto;
+using Core.Application;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace Functions.CreateAlarm
+{
+    public class CreateAlarm
+    {
+       private IAlarmService _alarmService;
+       public CreateAlarm(IAlarmService alarmService)
+       {
+           _alarmService = alarmService;
+       }
+
+       [FunctionName("CreateAlarm")]
+       public async Task<IActionResult> Run(
+           [HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = "alarms")] HttpRequest req,
+            ILogger log)
+            {
+                string reqBody = await new StreamReader(req.Body).ReadToEndAsync();
+                var alarm = JsonConvert.DeserializeObject<Alarm>(reqBody);
```

Comment:
Validere om alarmens attributter er korrekte. Er beskrivelsen tom eller er alarmkoden for eksemplet et negativ tal